code/modules/uplink/uplink_items/stealthy_tools.dm

```diff
@@ -26,8 +26,15 @@
 
 /datum/uplink_item/stealthy_tools/chameleon
 	name = "Chameleon Kit"
+//MONKESTATION EDIT START
+	/*
 	desc = "A set of items that contain chameleon technology allowing you to disguise as pretty much anything on the station, and more! \
 			Due to budget cuts, the shoes don't provide protection against slipping and skillchips are sold separately."
+	*/
+	desc = "A set of items that contain chameleon technology allowing you to disguise as pretty much anything on the station, and more! \
```

Comment:
Could just overwrite the desc in like `monkestation/code/modules/uplink/uplink_items/stealthy_tools.dm` instead

Reply:
done